src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java

```diff
@@ -216,6 +217,15 @@ public FFmpegBuilder addExtraArgs(String... values) {
     }
     return this;
   }
+  public FFmpegBuilder addHlsArgs(String... values) {
+    checkArgument(values.length > 0, "one or more values must be supplied");
+    checkNotEmpty(values[0], "first extra arg may not be empty");
+
+    for (String value : values) {
+      hls_args.add(checkNotNull(value));
+    }
+    return this;
+  }
```

Comment:
I don't see addHlsArgs called in any of the below tests. Is it needed?

I'd much rather have `addHlsPlaylistType`, `addHlsTime`, etc, be added as methods.  Or maybe a whole new add `addHls().PlaylistType(...)`